### PR TITLE
feat: support audio audio input  for kyc

### PR DESCRIPTION
**Pull request by kewde**
This PR fixes an issue where our KYC provider requires audio/mic capture.

### CODE REVIEW

**Review by ezenwankwogabriel — APPROVED** [review]
utACK

File: android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java [hunk 1]
```diff
@@ -1107,6 +1112,8 @@ public boolean onRequestPermissionsResult(int requestCode, String[] permissions,
                 if (grantResults[i] == PackageManager.PERMISSION_GRANTED) {
                   if (permissions[i].equals(Manifest.permission.CAMERA)) {
                     grantedPermissions.add(PermissionRequest.RESOURCE_VIDEO_CAPTURE);
+                  } else if (permissions[i].equals(Manifest.permission.RECORD_AUDIO)) {
```
Comment (kewde) [inline android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java:1115]: tbh in order for audio in the webview to work we need more than this permissions alone.

We need these, maybe we can remove `MODIFY_AUDIO_SETTINGS`, haven't checked:
```
    <uses-permission android:name="android.permission.RECORD_AUDIO"/>
    <uses-permission android:name="android.permission.MODIFY_AUDIO_SETTINGS" />
    <uses-permission android:name="android.permission.AUDIO_CAPTURE" />
```

**Review by andrejborstnik — APPROVED** [review]
utACK

**Review by akinncar — APPROVED** [review]
utACK

File: android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java (hunk 1 above)
Comment (davidfragalaureano) [reply android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java:1115]: Is `MODIFY_AUDIO_SETTINGS ` required too?

**Review by davidfragalaureano — APPROVED** [review]
utACK

**Comment by ezenwankwogabriel** [comment]
> MODIFY_AUDIO_SETTINGS

it's not required permission by `veriff`, not sure if it's required to use audio within the webview

**Pull request merged**